### PR TITLE
Add issue template for Calamari xcm onboarding of other parachains

### LINKED ISSUE #451: External XCM onboarding documentation
We need something similar to https://docs.google.com/document/d/1W8y00IcJb0JXPBF59aP4nm-c7DY8Ld02-yIAO7UxR80/edit to streamline the process of other projects testing XCM with us.
If we're going to rely on solely on Rococo then simply descrbe how to open channels, register assets, execute transfer.
If we're going to use our internal testnet (Baikal) for testnet then provide steps how to onboard a parachian as well.

Work in progress
- https://hackmd.io/Xs1A69biRP633MMQVeaNgQ

Remaining TODOs:
- [x] Dolphin endpoint
- [x] Dolphin Faucet
- [x] Time estimate for syncing to Rococo
- [x] Encoded call data for the xcm message to open HRMP channel
- [x] Choose common account that will be used for testing on all parachains
- [x] Add pictures
- [x] Create github issue template with instructions

### PR DESCRIPTION
**Pull request by ghzlatarev**
Signed-off-by: ghzlatarev <ghzlatarev@gmail.com>

## Description

closes: #451 

* Add an issue template to organize the process of testing XCM with other parachain teams and onboarding foreign assets on Calamari.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Wrote unit tests.
- [x] Updated relevant documentation in the code.
- [x] Added **one** line describing your change in `<branch>/CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [x] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [x] Verify benchmarks & weights have been updated for any modified runtime logics
- [x] If import a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- [x] If needed, update our Javascript/Typescript APIs. These APIs are offcially used by exchanges or community developers.
- [x] If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inhreited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+* Manta Team Contact:
+    - Georgi (XCM Eng): @Ghz (Telegram)
+    - Shumo (Co-founder, Tech.): @xstec (Telegram)
+
+## Local XCM Integration
+
+- As a first step we insist that both teams first run all tests on a local network.
+- For that you can download the latest manta binary from the Releases page.
+- Then use polkadot-launch to launch a `calamari-local` or `calamari-dev` network for testing.
```
Comment (Dengjianping) [inline .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md:33]: Perhaps we could provide a template of config file.

Comment (ghzlatarev) [reply .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md:33]: Yeah good idea, i'll just put it at the end and link to it.

**Comment by ghzlatarev** [comment]
@Garandor @stechu @Dengjianping this is ready for review.

File: .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    3. The destination has to be the relay chain, for dest (V1) set:
+    `{ parents:1, interior: Here }`
+    4. For the message (V2), you’ll be adding 4 items (described before):
+        1. `WithdrawAsset { id: Concrete { parents: 0, interior: Here}, Fungible: 1000000000000 }`
+        2. `BuyExecution { id: Concrete: {parents: 0, interior: Here}, Fungible: 1000000000000, weightLimit: Unlimited }`
+        3. `Transact { originType: Native, requireWeightAtMost: 1000000000, call: XcmDoubleEncoded: { encoded: RelayEncodedCallData } }`
+        
+		**Note:** you need to provide the encoded call data obtained before
+        
+		4. `RefundSurplus`
```
Comment (stechu) [inline .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md]: some formatting issue here?

File: .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+- Once this message is sent, the relay chain should execute the content and the request to open the channel with Dolphin
+- **Please let us know once you’ve requested opening the channel because the request needs to be accepted by Dolphin.**
+
+Here's an example of the fully formed extrinsic:
+
+![https://i.imgur.com/GUN8qJd.png](https://i.imgur.com/GUN8qJd.png)
+
+![https://i.imgur.com/3ONY21d.png](https://i.imgur.com/3ONY21d.png)
+
+## Accepting HRMP Channel with Calamari
```
Comment (stechu) [inline .github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md]: Should here be Dolphin?

**Review by stechu — APPROVED** [review]
LGTM. Find some minor issues.

**Comment by ghzlatarev** [comment]
> LGTM. Find some minor issues.

Thanks, resolved.

**Pull request merged**